sg_covid_impact/google_footfall.py

```diff
@@ -0,0 +1,53 @@
+import requests
+import logging
+from zipfile import ZipFile
+from io import BytesIO
+import os
+import pandas as pd
+import sg_covid_impact
+
+project_dir = sg_covid_impact.project_dir
+
+_GOOG_URL = "https://www.gstatic.com/covid19/mobility/Region_Mobility_Report_CSVs.zip"
+_TARGET_DIR = f"{project_dir}/data/raw/google_footfall"
+_COUNTRY = "GB"
+
+
+def fetch_footfall_data():
+    """Collects Google footfall data"""
+    logging.info("Fetching Google footfall data")
+    reg_g = requests.get(_GOOG_URL)
+    z = ZipFile(BytesIO(reg_g.content))
+    z.extractall(_TARGET_DIR)
+
+
+def process_footfall_data(country_code):
+    """Reads and processes country data for a country
+    Args:
+        country_code (str): country iso code
+        TODO: adapt to extract subregional data! 
```

Comment:
Is this required to fulfill the PR?

Reply:
No - for now we are just focusing on national trends + sub-national data only seems to be available for a subset of LADs. I would create a new issue for it.